rustls-platform-verifier/Cargo.toml

```diff
@@ -38,16 +38,16 @@ paste = { version = "1.0", default-features = false, optional = true } # Only us
 
 [target.'cfg(all(unix, not(target_os = "android"), not(target_vendor = "apple"), not(target_arch = "wasm32")))'.dependencies]
 rustls-native-certs = "0.8"
-webpki = { package = "rustls-webpki", version = "0.102", default-features = false }
+webpki = { package = "rustls-webpki", version = "0.103", default-features = false }
 
 [target.'cfg(target_os = "android")'.dependencies]
 rustls-platform-verifier-android = { path = "../android-release-support", version = "0.1.0" }
 jni = { version = "0.21", default-features = false }
-webpki = { package = "rustls-webpki", version = "0.102", default-features = false }
-android_logger = { version = "0.13", optional = true } # Only used during testing.
+webpki = { package = "rustls-webpki", version = "0.103", default-features = false }
```

Comment:
Is it worth updating this one if rustls is still using the older version anyway? I am slightly worried about error variants not lining up.

Reply:
rustls is now using the newer one IIRC?

Reply:
At least in the PR diff it looks like `rustls`, which wasn't upgraded here, is still using the older version:
<img width="684" alt="image" src="https://github.com/user-attachments/assets/fdae0318-1900-4dd2-8057-1b13a8ea9d30" />

Reply:
Ah yeah, we merged it upstream but have not released it yet.